apps/api/src/app/workflows-v2/util/url-utils.ts

```diff
@@ -0,0 +1,6 @@
+export function isValidUrlForActionButton(url: string): boolean {
```

Comment:
i think we can be more generic here as it is used for `redirect` object as well.
```suggestion
export function isValidRedirectUrl(url: string): boolean {
```

Reply:
merge redirect object into action